### PR TITLE
Handle errors in release lane

### PR DESCRIPTION
**Pull request by neodym**
## Description
This will actually post errors/issue we have during release on our slack channel and inform us about failing releases.

### CODE REVIEW

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/0b7ef5c8-8ff5-4cce-9d23-ca1eae8edf69" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>WeTransferPRLinterTests: Executed 18 tests (0 failed, 0 retried, 0 skipped) in 16.929 seconds</td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |
WeTransferPRLinter | 0.00% | ⚠️
WeTransferPRLinter | 92.19% | ✅

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against ef09c7142e05708b47651c4f72636d708a478809
</p>

**Review by BasThomas — APPROVED** [review]

File: Fastlane/deployment_lanes.rb [hunk 1]
```diff
@@ -292,8 +292,10 @@
     success_message = "A new #{release_type} has been submitted to the App Store."
     UI.success "#{success_message} (#{tag_name})"
     slack_message(success_message, type: :submitted, tag_name: tag_name, release_url: release_url)
-  rescue StandardError => e
-    UI.error e
+
+  rescue StandardError => exception
+    UI.error exception
+    handle_error(:release, exception)
```
Comment (amirdew) [inline Fastlane/deployment_lanes.rb:298]: Does this method handle removing tags and ...?

**Review by amirdew — APPROVED** [review]

File: Fastlane/deployment_lanes.rb (hunk 1 above)
Comment (neodym) [reply Fastlane/deployment_lanes.rb:298]: Not yet? Should it? I guess that would be nice.. we shall create a ticket for that :)

Comment (amirdew) [reply Fastlane/deployment_lanes.rb:298]: Yeah, I think it should, because if we run the workflow again we may have conflicts

Comment (neodym) [reply Fastlane/deployment_lanes.rb:298]: @peagasilva
Could we use `clean_up_release_from_tag` here?
At the moment `handle_error` only calls it, if the lane is `release_from_tag`.

Comment (peagasilva) [reply Fastlane/deployment_lanes.rb:298]: @neodym I would say so - we need to only update the filter. Sorry for the delay, I totally missed this message.

**Pull request merged**